Makefile

```
@@ -2,6 +2,8 @@ NAME=incident-commander
 OS   = $(shell uname -s | tr '[:upper:]' '[:lower:]')
 ARCH = $(shell uname -m | sed 's/x86_64/amd64/')
 DATE = $(shell date  "+%Y-%m-%d %H:%M:%S")
+GIT_COMMIT = $(shell git rev-parse HEAD 2>/dev/null || echo none)
```

checkmake (0.3.2)

[warning] Variable "DATE" possibly contains a timestamp and should be simply expanded.

(timestampexpanded)